src/Surfnet/ServiceProviderDashboard/Application/Command/Service/CreateServiceCommand.php

```diff
@@ -30,160 +32,112 @@ class CreateServiceCommand implements Command
 {
     /**
      * @var string
-     * @Assert\Uuid(strict=false)
      */
+    #[Assert\Uuid(strict: false)]
     private $guid;
 
     /**
-     * @var string
-     * @Assert\NotBlank
-     * @SpDashboardAssert\UniqueTeamName()
+     * @var                                string
      */
+    #[SpDashboardAssert\UniqueTeamName()]
+    #[Assert\NotBlank]
     private $teamName;
 
     /**
      * @var string
-     * @Assert\NotBlank
      */
+    #[Assert\NotBlank]
     private $name;
 
-    /**
-     * @var string
-     * @Assert\NotBlank
-     * @Assert\Email()
-     */
-    private $teamManagerEmail;
+    #[Assert\NotBlank]
+    #[Assert\Email]
+    private ?string $teamManagerEmail = null;
 
     /**
      * @var string
      */
```

Comment:
Why not add type definition to this field?

Reply:
unfortunatly, the typing is not consistent enough to trust the phpdoc to apply it to the vars. Therefor, a lot of vars arent' still typed.